packages/sit-onyx/src/components/OnyxTab/OnyxTab.vue

```diff
@@ -0,0 +1,117 @@
+<script lang="ts" setup>
+import { computed, inject } from "vue";
+import { useDensity } from "../../composables/density";
+import { TABS_INJECTION_KEY } from "../OnyxTabs/types";
+import type { OnyxTabProps } from "./types";
+
+const props = defineProps<OnyxTabProps>();
+
+defineSlots<{
+  /**
+   * Tab panel / content.
+   */
+  default(): unknown;
+  /**
+   * Optional slot to override the tab content. By default, the `label` property will be displayed.
+   */
+  tab?(): unknown;
+}>();
+
+const { densityClass } = useDensity(props);
+const tabsContext = inject(TABS_INJECTION_KEY);
+
+const isActive = computed(() => tabsContext?.headless.state.isVisible.value(props.value) ?? false);
```

Comment:
IMHO, the template would be cleaner, if we use computed to extract the elements.
Also, it seems like the `isVisible` function is redundant, when we use ´tab.["aria-selected"]´
```suggestion
const tab = computed(() => tabsContext?.headless.elements.tab.value({ value: props.value }));
const tabpanel = computed(() =>
  tabsContext?.headless.elements.tabpanel.value({ value: props.value }),
);
const isActive = computed(() => tab.value?.["aria-selected"] ?? false);
```